client/src/app.jsx

```diff
@@ -17,7 +18,7 @@ class App extends React.Component {
 
   render () {
     return (
-    <div class="container">
+    <div className="container">
```

Comment:
Please use less generic classnames UNLESS you are going to convert them using some babel magic